concepticondata/concepticon.tsv

```diff
@@ -1348,7 +1348,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1347	NAPE (OF NECK)	The body	The back part of the neck.	Person/Thing	
 1348	BOAR (MALE PIG)	Animals	An adult male pig.	Person/Thing	
 1349	CHURCH	Religion and belief	A building where Christian religious activities take place.	Person/Thing	
-1350	MONKEY	Animals	"A primate from the group ""New World monkeys"" or ""Old World monkeys"" (Simiiformes, excluding the superfamily Hominoidea or apes) that lives mainly in rainforests and is distinguished from an Ape by its smaller size and its tail."	Person/Thing	
+1350	MONKEY	Animals	A primate from the group "New World monkeys" or "Old World monkeys" (Simiiformes, excluding the superfamily Hominoidea or apes) that lives mainly in rainforests and is distinguished from an Ape by its smaller size and its tail.	Person/Thing	
```

Comment:
again, deleting `"` completely is what I suggest. One can use markdown and write `*New world monkeys*` instead.